### ISSUE TITLE
Would love to see actual full description of fq and efq 

### ISSUE THREAD

**Issue by aredridel** [bloomreach/docs-feedback#27]
https://documentation.bloomreach.com/search-and-merchandising/docs/milestone-4.-api-deployment/search-and-merchandising-feature-apis/product-search-api/use-case-dependent/use-case-dependent-parameters.html

The examples almost entirely overlap, so the difference is unclear, and what the syntax is for either is not fully specified.

**Comment by pemasp** [comment]
Hello @aredridel 

Thank you for your feedback. We are currently working on this and will update the documentation soon.

**Comment by aredridel** [comment]
Thank you!

**Comment by pemasp** [comment]
@aredridel While we work on adding detailed examples, I wanted to share the **Faceting and Filtering** page that has more information on using the fq and efq parameters. 

Here the link to the page: https://documentation.bloomreach.com/search-and-merchandising/docs/milestone-4.-api-deployment/search-use-cases/faceting-and-filtering.html

Thank you for your patience.

**Comment by aredridel** [comment]
Yep! that's what I've been using. In particular, it's ambiguous on what the difference _is_ and what the limits of them are; There are no examples of queries with ranges — just responses. There is no documentation for how the query parameters are encoded with respect to spaces and other URL-special characters. It's not documented how top level `fq` are combined — and? or? Magical ranking sauce?

It's also fascinating the approach taken, as string concatenation as the way to document these parameters, rather than mapping parts in a grammar. It means that everything has to be evaluated in prior context rather than granules of knowledge that can be combined.

Does order matter? That's also not documented.

When the responses come back, is their order significant?
